ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -353,20 +353,26 @@ let signature_help (state : State.t)
   let open Fiber.Result.O in
   let* doc = Fiber.return (Document_store.get store uri) in
   let pos = Position.logical position in
+  let prefix =
+    Compl.prefix_of_position (Document.source doc) pos ~short_path:false
```

Comment:
Why `~short_path:false`?

Reply:
It doesn't make a different either way. Should I make it `true` instead?

Reply:
I'd leave a comment explaining that it doesn't make a difference. `short_path:true` is a little faster.